service/user-write/bin/user_snapshotter.php

```diff
@@ -13,32 +13,39 @@
 namespace Prooph\MicroDo\UserWrite\Script;
 
 use Prooph\Common\Messaging\Message;
+use Prooph\MicroDo\UserWrite\Infrastructure\UserAggregateDefinition;
 use Prooph\EventStore\EventStore;
+use Prooph\EventStore\Pdo\Projection\PostgresProjectionManager;
 use Prooph\Micro\SnapshotReadModel;
-use Prooph\MicroDo\UserWrite\Infrastructure\UserAggregateDefinition;
+use Prooph\MongoDb\SnapshotStore\MongoDbSnapshotStore;
 
 $autoloader = require __DIR__ . '/../vendor/autoload.php';
 
 $factories = include __DIR__ . '/../src/Infrastructure/factories.php';
 
 $eventStore = $factories['eventStore']();
-/* @var EventStore $eventStore */
-
+$pdoConnection = $factories['pdoConnection']();
 $aggregateDefinition = new UserAggregateDefinition();
+$mongoClient = $factories['mongoConnection']()->client();
+
+/* @var EventStore $eventStore */
 
 $readModel = new SnapshotReadModel(
     $factories['snapshotStore'](),
     new UserAggregateDefinition()
```

Comment:
how does this consume UserAggregateDefinition https://github.com/prooph/snapshotter/blob/develop/src/SnapshotReadModel.php#L54-L56